parity-util-mem/src/allocators.rs

```diff
@@ -26,6 +26,11 @@
 //!	 - jemalloc: use jemallocator crate
 //!	 - weealloc: default to `estimate_size`
 //!	 - dlmalloc: default to `estimate_size`
+//! - arch x86/macos:
+//!	 - no features: use default alloc, requires using `estimate_size`
```

Comment:
why does it require `estimate_size`, isn't `MallocSizeOf` supposed to work on macOS?

Reply:
it is the building failure that I did fail to see in previous commit (test was not activated in ci), it did not find malloc_size_of symbol (or jemalloc if prefixed), I would like to find a way to call the default allocator size of primitive but I do not know any.

Reply:
shouldn't we use `unprefixed_malloc_on_supported_platforms` on [jemallocator](https://github.com/alexcrichton/jemallocator/tree/master/jemalloc-sys#cargo-features)?